front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -279,6 +273,11 @@ const IdeasWithFiltersSidebar = ({
               />
             </>
           )}
+          <Text mb="8px">
```

Comment:
We only want this to be shown if there is no specific "inputTerm" set though, since on the Project page we include the count in the Title instead :)

E.g. Now we see this on project pages:
![image](https://github.com/user-attachments/assets/0ec1c1a7-3dd5-40c2-9a3e-a8b971955fc2)

Reply:
Oops, thanks for catching. I'll add the check. Actually, this should live on the IdeasIndexPage if it's only needed for the URL, but lots more refactoring needed for that. This code is a mess 🙃

Reply:
Changed!